tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/graphql/extensions/TaskExtension.java

```diff
@@ -14,26 +14,34 @@
  * SUBJECT AS SET OUT BELOW, THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  * NOTHING IN THIS AGREEMENT EXCLUDES OR RESTRICTS A PARTY’S LIABILITY FOR (A) DEATH OR PERSONAL INJURY CAUSED BY THAT PARTY’S NEGLIGENCE, (B) FRAUD, OR (C) ANY OTHER LIABILITY TO THE EXTENT THAT IT CANNOT BE LAWFULLY EXCLUDED OR RESTRICTED.
  */
-package io.camunda.tasklist.webapp.graphql.query;
+package io.camunda.tasklist.webapp.graphql.extensions;
 
-import graphql.kickstart.tools.GraphQLQueryResolver;
+import static io.camunda.tasklist.util.SpringContextHolder.getBean;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import graphql.annotations.annotationTypes.GraphQLTypeExtension;
+import graphql.schema.DataFetchingEnvironment;
 import io.camunda.tasklist.webapp.graphql.entity.TaskDTO;
-import io.camunda.tasklist.webapp.graphql.entity.TaskQueryDTO;
-import io.camunda.tasklist.webapp.service.TaskService;
-import java.util.List;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
+import io.camunda.tasklist.webapp.mapper.TaskMapper;
 
-@Component
-public final class TaskQueryResolver implements GraphQLQueryResolver {
+@GraphQLTypeExtension(TaskDTO.class)
+public class TaskExtension {
 
-  @Autowired private TaskService taskService;
+  @GraphQLField
+  @GraphQLNonNull
+  public static String processName(final DataFetchingEnvironment env) {
+    return getBean(TaskMapper.class).getProcessName(env.getSource());
+  }
 
-  public List<TaskDTO> tasks(TaskQueryDTO query) {
-    return taskService.getTasks(query);
+  @GraphQLField
+  @GraphQLNonNull
+  public static String name(final DataFetchingEnvironment env) {
```

Comment:
One question, not clear for me why this method calls `name`, is any specific reason? (was thinking on getName, but maybe I am missing something)